_usage/locales.md

```diff
@@ -49,59 +49,59 @@ that describe the collation attributes, as shown in the table below:
 | CS          | Case sensitive                                         |
 | PREF        | To sort uppercase letters before lowercase letters, use a PREF collation. If comparison is case-insensitive, the uppercase version of a letter sorts before the lowercase version, if there is no other distinction. The ICU library supports uppercase preference with `colCaseFirst=upper`, but not for `CI_AS` collations. PREF can be applied only to `CS_AS` - deterministic.  |
 
-PostgreSQL doesn\'t support the `LIKE` clause on non-deterministic
+PostgreSQL doesn't support the `LIKE` clause on non-deterministic
 collations, but Babelfish supports it for `CI_AS` collations.
-Babelfish doesn\'t support `LIKE` on AI collations. Pattern matching
+Babelfish doesn't support `LIKE` on AI collations. Pattern matching
 operations on non-deterministic collations are also not supported.
 
 To establish Babelfish collation behavior, set the following
 parameters :
 
 | Parameter             | Description                                   |
 | --------------------- | --------------------------------------------- |
-| `default_locale`      | The ` default_locale ` parameter is used in combination with the collation attributes in the table above to customize collations for a specific language and region. The default value is `en-US`. The default locale applies to all Babelfish collations and to all SQL Server collations that are mapped to Babelfish collations. This parameter may be changed after initial Babelfish database creation time, but it won\'t affect the locale of existing collations.            |
-| `server_collation_name`  | The collation used as the default collation at both the server level and the database level. The default value is `sql_latin1_general_cp1_ci_as`. You can choose from the collations in the table that follows for the Collation name field when you create your PostgreSQL cluster for use with Babelfish. Don\'t modify the `server_collation_name` after the Babelfish database is created.  |
+| `default_locale`      | The `default_locale` parameter is used in combination with the collation attributes in the table above to customize collations for a specific language and region. The default value is `en-US`. The default locale applies to all Babelfish collations and to all SQL Server collations that are mapped to Babelfish collations. This parameter may be changed after initial Babelfish database creation time, but it won't affect the locale of existing collations.            |
+| `server_collation_name`  | The collation used as the default collation at both the server level and the database level. The default value is `sql_latin1_general_cp1_ci_as`. When you create your PostgreSQL cluster for use with Babelfish, you can choose the &ldquo;Collation name&rdquo; from the following table. Don't modify `server_collation_name` after the Babelfish database is created!  |
```

Comment:
I'd replace the '!' with a '.'

Reply:
Changed in 1ae7fb6d8b9f64b2355e0839f9c958e739a62c9c.